v2/README.md

```diff
@@ -61,7 +61,7 @@ Sample YAMLs for creating each of these resources can be found in the [samples d
    ```
 3. Download [the latest **v2+** release](https://github.com/Azure/azure-service-operator/releases) of Azure Service Operator and install it into your cluster.
    ```bash
-   kubectl apply -f azureserviceoperator_v2.0.0-alpha.0.yaml
+   kubectl create -f azureserviceoperator_v2.0.0-alpha.0.yaml
```

Comment:
Do you need to explain why to use `create` and not `apply`?

Reply:
I don't think so here - it would be a distraction in the installation instructions. If a user who's a bit more knowledgeable tries using `apply` instead they'll see why we're saying to use `create` - in that case if they then go back to using `create` things will work fine (although they'll get errors that some things already exist).